src/libraries/System.IO.Ports/src/System.IO.Ports.csproj

```diff
@@ -25,6 +25,7 @@
     <Compile Include="System\IO\Ports\StopBits.cs" />
   </ItemGroup>
   <ItemGroup Condition="'$(TargetsNetFx)' != 'true' and '$(TargetsWindows)' == 'true'">
+    <Compile Include="$(CoreLibSharedDir)System\Diagnostics\CodeAnalysis\NullableAttributes.cs" Link="System\Diagnostics\CodeAnalysis\NullableAttributes.cs" />
```

Comment:
As mentioned in comment: https://github.com/dotnet/runtime/pull/237#discussion_r352785719 we can just add `<Nullable>annotations</Nullable` to this project if you like that approach best.

Reply:
Hmm, ok, I'll switch to that. Thanks.